.storybook/head.html

```diff
@@ -0,0 +1,12 @@
+<script type="text/javascript" src="//fast.fonts.net/jsapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.js"></script>
+<script type="text/javascript" src="https://fast.fonts.net/jsapi/core/mt_otf.js"></script>
+
+<style>
+  @font-face {
+	font-family: 'artsy-icons';
+	src: url("/fonts/artsy-icons.eot?uo9ko");
+	src: url("/fonts/artsy-icons.eot?#iefixuo9ko") format('embedded-opentype'), url("/fonts/artsy-icons.woff2?uo9ko") format('woff2'), url("/fonts/artsy-icons.ttf?uo9ko") format('truetype'), url("/fonts/artsy-icons.woff?uo9ko") format('woff'), url("/fonts/artsy-icons.svg?uo9ko#artsy-icons") format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+</style>
```

Comment:
Instead of this separate head file, you can just insert it into the `iframe.html` file inside the `.storybook` directory.

Although I do wonder what the better styled-components way is. Q stick search turned up this https://github.com/styled-components/styled-components/blob/3e5d63fd71ad06c37cfb5fe6bc8e77553ddc0fb2/docs/css-we-support.md, I’m not entirely sure in what component you’re supposed to do that, maybe any component that refers to the font and they deduce it before actually inserting into the `<head>`?